simulation-runner/src/main/scala/com/puppetlabs/gatling/node_simulations/PerfTestLarge.scala

```diff
@@ -20,6 +20,10 @@ class PerfTestLarge extends SimulationWithScenario {
                          Map("reportFile" -> "IntentionalChangeReport_0044_request.txt")
                        ).circular
 
+	val facts_str  = scala.io.Source.fromFile("user-files/facts/IrvingFactsChange_facts_raw.txt").mkString
+	val facts_fdr  = csv("user-files/facts/IrvingFactsChange_custom_facts.csv")
+	val hostf_fdr  = csv("user-files/facts/IrvingFactsChange_custom_hostf.csv").circular
```

Comment:
these variable names are hard for me to read. i actually thought this was a duplicate assignment